### ISSUE TITLE
Error appear by clicking on "Horizontal Split", "Vertical Split","Inside Pager" under QtConsole application.

### ISSUE THREAD

**Issue by Usama23** [jupyter/qtconsole#179]
OS: Windows
Anaconda Navigator 1.4.rc1

**Steps to Reproduce:**
   1: Login to Anaconda Navigator.
   2:  Under home tab, Launch QtConsole application.
   3: In application, select "view" from menu given on the top.
   4: From the drop down of View select pager, and then under the pager tab menu, click to open 
      "Horizontal split","Vertical Split","Inside Pager" one by one.

**Actual Result:**
     When click on these three menu options to open, one by one, pop up error window appear.(All these
      three have same type errors).

**Expected Result:**
   By clicking on these options, related action should perform successfully.

![screenshot_31](https://cloud.githubusercontent.com/assets/21064099/21720420/cda5e4e2-d444-11e6-9719-26f0675b1618.png)

![screenshot_30](https://cloud.githubusercontent.com/assets/21064099/21720175/931f0bf6-d443-11e6-862b-c068d2975f62.png)

**Comment by ccordoba12** [comment]
Thanks @Usama23 for moving this issue here. @Usama23 does QA for Continuum :-)

This seems a simple error. @mariacamilaremolinagutierrez, please work on it.

**Comment by Usama23** [comment]
@ccordoba12 Pleasure :)

**Comment by Usama23** [comment]
@ccordoba12  can you please mention"Glueviz" issue report GH link if any? like this qtconsole or spyder.

**Comment by ccordoba12** [comment]
Sure, this is the repo for Gueviz:

https://github.com/glue-viz/glue

**Comment by mariacamilaremolinagutierrez** [comment]
@ccordoba12 How is this supposed to look? When I run ```python -m qtconsole --paging=vsplit``` or hsplit, I get the same output as if there was not paging. I do not get the error but even after reselecting the option or choosing another one, I still have only one window, it doesn't split. I can fix getting the error but I don't know how it should behave after selecting the option.

**Comment by ccordoba12** [comment]
@mariacamilaremolinagutierrez, I could reproduce this error following exactly the same steps mentioned by @Usama23. Please start `qtconsole` like `python -m qtconsole` on Windows to do it.

Initially the idea is just to prevent that ugly error dialog from displaying, then we'll if paging is working or not.

**Comment by mariacamilaremolinagutierrez** [comment]
@ccordoba12 Yes I did get the same error, and I can avoid it already. My question is what should be seen instead? (So that I make it follow the respective behaviour) Because right now paging alone is doing nothing.

**Comment by davvid** [comment]
using `ipython qtconsole --paging=vsplit` if I use the `Help -> Qt Console` menu, the help text is displayed in a split "pager", and that pager does seem to be affected by this menu.

I think some folks (myself included -- it's how I found this issue) might expect that the "split" mode would redirect the output of the python execution into the split pager as well, but that stays inline.. so that might explain your question @mariacamilaremolinagutierrez 

(BTW, if there is a way to get the actual output to go into the split pager, please let me know how to do that since I was looking for that feature and stumbled upon this issue along the way)

**Comment by takluyver** [comment]
> if there is a way to get the actual output to go into the split pager

I don't think there's a way to send output there directly, but `IPython.core.page.page()` should be able to send a string to the pager. You could combine that with `IPython.utils.capture.capture_output` to send output to the pager.
